client/js/splitpanelsection.js

```diff
@@ -0,0 +1,625 @@
+
+var _ = require('underscore')
+var $ = require('jquery')
+
+var SplitPanelSection = function(index, $panel, initData, parent) {
+
+    this.parent = parent;
+    this.listIndex = index;
+    this.name = $panel.attr("id");
+
+    if (_.isUndefined(this.name))
+        throw "All splitter panels require an id attribute";
+
+    $panel.css("position", "absolute");
+    $panel.css("box-sizing", "border-box");
+
+    this.panel = $panel;
```

Comment:
a good convention is to always refer to jQuery elements with the $, even when they're attached to a this. so here i would go:

    this.$panel = $panel